conferences/2018/android.json

```diff
@@ -1,4 +1,58 @@
 [
+  {
+    "name": "Devoxx",
```

Comment:
Should we name this `Devoxx Belgium` since there are multiple Devoxx conferences? cc @prigara @trivikr

Reply:
I usually keep the conference name same (like Smashing Conference)
Users can view location just below the conf name